public/js/redux/index.js

```diff
@@ -1,18 +1,18 @@
 import { combineReducers } from 'redux';
 import { routerReducer } from 'react-router-redux';
 import configureStore from './configureStore';
-import chartsRedux from './chartsRedux';
-import updateFilterRedux from './updateFilterRedux';
-import updateProgressRedux from './updateProgressRedux';
-import commissioningDesksRedux from './commissioningDesksRedux';
+import chartsReducer from 'reducers/chartsReducer';
+import updateFilterReducer from 'reducers/updateFilterReducer';
+import updateProgressReducer from 'reducers/updateProgressReducer';
+import commissioningDesksReducer from 'reducers/commissioningDesksReducer';
 
 export default () => {
     /* ------------- Assemble The Reducers ------------- */
     const rootReducer = combineReducers({
-        filterVals: updateFilterRedux,
-        charts: chartsRedux,
-        isUpdating: updateProgressRedux,
-        commissioningDesks: commissioningDesksRedux,
+        filterVals: updateFilterReducer,
```

Comment:
I'd just export these as `filterVals, chartsz` etc. The filenames work to define them as reducers